src/controller/CHIPDeviceController.cpp

```diff
@@ -754,6 +760,8 @@ CHIP_ERROR DeviceCommissioner::Init(CommissionerInitParams params)
     ReturnErrorOnFailure(mIDAllocator.ReserveUpTo(nextKeyID));
     mPairingDelegate = params.pairingDelegate;
 
+    SetDeviceAttestationVerifier(Examples::GetExampleDACVerifier());
```

Comment:
This should be in the examples themselves (chip-tool, etc)

Reply:
will update